docs/cluster-deployment-guide.md

```diff
@@ -363,7 +363,15 @@ If you are using a version that is older than 1.0.0, reach out to us at [communi
 First check to see if an upgrade to the database schema is required for the version you wish to upgrade to.
```

Comment:
The info box above ("If you are using a version that is older than 1.0.0...") doesn't render with enough of a standoff/gap from this paragraph. In the rendered version it looks like this paragraph ("First check...") only applies to older than 1.0.0, which is confusing.

Maybe we should just remove the info box? There can't be anyone still using < 1.0.0, right?